### DISCUSSION TITLE
Transfer files to portapack (.C16 file)

### DISCUSSION THREAD

**Discussion by stenbus** [jopohl/urh#984] [Q&A] [answered]
Hi!
I've been sitting at home recording signals with my hackrf. Now that the signals are analyzed and ready to be used, I'm having trouble uploading them tho the SD card on the portapack. the files URH saves as is complex16s but the portapack requires C16 files and a TXT file alongside. Do any of you have an idea on how to do this the easy way? Bringing my whole setup out in the field is to much hassle. 

thanks!

**Comment by jimilinuxguy** [comment, marked as answer]
@stenbus checkout the bottom of the wiki on Mayhem

This GRC script can be found at [firmware/tools/convert_C16_to_complex.grc](https://github.com/eried/portapack-mayhem/blob/next/firmware/tools/convert_C16_to_complex.grc)

**Comment by dpugliese6** [comment]
Changing the file extension from complex32s to C16 (or complex16s to C8) worked for me. As for the .txt file, you can create it manually.
